<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>791</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft JhengHei UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Grading</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>621</width>
      <height>731</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0" colspan="2">
      <widget class="QTableWidget" name="sheetEdit">
       <property name="toolTip">
        <string/>
       </property>
       <property name="rowCount">
        <number>40</number>
       </property>
       <property name="columnCount">
        <number>10</number>
       </property>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
      </widget>
     </item>
     <item row="5" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="scoreLabel">
         <property name="font">
          <font>
           <family>Microsoft JhengHei UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>成绩数量：</string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="scoreCount_label">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="gradeLable">
         <property name="font">
          <font>
           <family>Microsoft JhengHei UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>等级数量：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="gradeCount_label">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QLabel" name="info_lable">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>示例表格-(2,2)~(2,2)-(3,2)~(3,2)-男子一千米中考评分</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="markScoreButton">
       <property name="font">
        <font>
         <family>Microsoft JhengHei UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>标记为成绩区域</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="editingLabel">
         <property name="font">
          <font>
           <family>Microsoft JhengHei Light</family>
           <pointsize>22</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>正在编辑：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="xlsTitle_lable">
         <property name="font">
          <font>
           <family>Microsoft JhengHei UI</family>
           <pointsize>22</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>示例表格</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="convertCheckBox">
         <property name="toolTip">
          <string>启用后可以在成绩中使用以下符号：
分符号′  秒符号″  时符号:
分简写min 秒简写s 时简写h</string>
         </property>
         <property name="text">
          <string>时间表示法自动转换</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="rulesButton">
         <property name="font">
          <font>
           <family>Microsoft JhengHei</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>规则</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="writeInButton">
         <property name="font">
          <font>
           <family>Microsoft JhengHei</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>写入</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="font">
          <font>
           <family>Microsoft JhengHei</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>保存</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="markGradeButton">
       <property name="font">
        <font>
         <family>Microsoft JhengHei UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>标记为等级区域</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>664</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionopenfile">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
